src/router.js

```diff
@@ -30,11 +30,14 @@ export const mountRoutes = (app, provider) => {
       const notifications = messages[req.query.notification]
         ? [messages[req.query.notification]]
         : [];
-
+ 
+      const acr = req.session.infos.acr_values ? req.session.infos.acr_values : '';
```

Comment:
This will still crash if infos is not defined...

Reply:
handled, but you commented before i pushed a new version